packages/grafana-ui/src/components/DateTimePickers/TimeZonePicker/TimeZoneOption.tsx

```diff
@@ -26,7 +26,7 @@ export interface SelectableZone extends SelectableValue<string> {
   searchIndex: string;
 }
 
-export const WideTimeZoneOption: React.FC<PropsWithChildren<Props>> = (props, ref) => {
+export const WideTimeZoneOption = (props: PropsWithChildren<Props>) => {
```

Comment:
I've not come across `PropsWithChildren` before. Is it something we want to keep? (Not suggesting removing it in this PR, more asking for my own understanding.)

Reply:
We use it in a few places... it is just a buildin react interface that has children specified